resources/test_data.json

```diff
@@ -467,5 +467,93 @@
         "note": "",
         "bug_type": "crash",
         "checker_qual_required": false
+    },
+    {
+        "issue_id" : "cf-3025",
+        "url": "https://github.com/tahiat/guava.git",
+        "branch": "cf-3020",
+        "commit_hash": "",
+        "project_name": "guava",
+        "build_command": "",
+        "root_dir": "guava/src/", 
+        "targets": [
+            {
+                "method": "difference(Map<? extends K, ? extends V>, Map<? extends K, ? extends V>, Equivalence<? super V>)",
+                "file": "Maps.java",
+                "package": "com.google.common.collect",
+                "inner_class": ""
+            }
+        ],
+        "cf_version": "3.0.1",
+        "java_version": "8",
+        "note": "",
+        "bug_type": "semi_crash",
+        "bug_pattern": {
+            "file_pattern": "Compilation unit: (.*)",
+            "error_pattern": "error: (.+?):",
+            "exception_pattern": "Exception: (.+):",
+            "stack_trace_pattern": "Stack trace: (.+)"
+        },
+        "checker_qual_required": false
+    },
+    {
+        "issue_id" : "jdk-8269121",
+        "url": "https://github.com/tahiat/TypeInferenceBugOfMethodReferences.git",
+        "issue_url": "https://bugs.openjdk.org/browse/JDK-8269121",
```

Comment:
This doesn't look like a compile-time crash that we can use Specimin to detect (it's caused by incorrect code gen). I assume this is the one that you mentioned in our meeting a few minutes ago @tahiat?

Reply:
Prof, this is the issue I mentioned as runtime issue.  This mail thread discusses how to reproduces the issue. 
https://mail.openjdk.org/pipermail/compiler-dev/2021-June/017341.html. Issue reproduces when below step is followed
mvn clean compile.  --> compiles the classes
java -classpath target/classes/ net.mirwaldt.bug.Main -->> exception occurs in this stage.

Reply:
Yes professor, your are right. This is the issue i mentioned.